ProjectLighthouse/Types/Profiles/Pins.cs

```diff
@@ -0,0 +1,12 @@
+using LBPUnion.ProjectLighthouse.Serialization;
+
+namespace LBPUnion.ProjectLighthouse.Types.Profiles
+{
+    public class Pins
+    {
+        public long[] progress { get; set; }
+        public long[] awards { get; set; }
+        public long[] profile_pins { get; set; }
```

Comment:
You can keep the naming style of the JSON data while also keeping the naming style of C# using a `JsonPropertyName` attribute.
Heres a (modified) example [from the Microsoft wiki](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-customize-properties?pivots=dotnet-5-0#customize-individual-property-names):
```csharp
[JsonPropertyName("wind_speed")]
public int WindSpeed { get; set; }
```